src/Stryker.Core/Stryker.Core.UnitTest/StrykerOptionsTests.cs

```diff
@@ -32,7 +32,7 @@ public void Constructor_WithIncorrectLoglevelArgument_ShouldThrowStrykerInputExc
 
             var ex = Assert.Throws<StrykerInputException>(() =>
             {
-                new StrykerOptions("c:/test", "Console", "", 0, null, logLevel, false, 1, 80, 60, 0);
+                new StrykerOptions(logLevel: logLevel);
```

Comment:
Different basepath due to default values.

Reply:
Does it really meter here?